src/system.h

```diff
@@ -82,7 +82,9 @@ GAP_STATIC_ASSERT(sizeof(void *) == SIZEOF_VOID_P, "sizeof(void *) is wrong");
 **  at least GAP won't be the only program to run into problems.
 */
 enum {
-#ifdef PATH_MAX
+// The 'PATH_MAX != 4096' test stops compilers warning about idential values
+// on both sides of the : in the line below
+#if defined(PATH_MAX) && PATH_MAX != 4096
```

Comment:
Hmm, can't we then go one step further and do this?
```C
#if defined(PATH_MAX) && PATH_MAX > 4096
    GAP_PATH_MAX = PATH_MAX,
#else
    GAP_PATH_MAX = 4096,
#endif
```

Reply:
That's much nicer! I'll do that.